testsuite/shared/enable-elytron.cli

```diff
@@ -0,0 +1,40 @@
+embed-server --server-config=standalone.xml
```

Comment:
Yes, there is. 

Both CLI scripts enable Elytron across the server in the first place. And the `modify-elytron-*` adds some more steps to change Elytron configuration itself.